libs/dh/market-participant/feature-edit-gridarea/src/lib/market-participant-edit-gridarea/dh-market-participant-edit-gridarea.component.html

```diff
@@ -0,0 +1,57 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ng-container *transloco="let t; read: 'marketParticipant.organization.create'">
```

Comment:
This will be part of a follow-up PR, this was just to get the inital grid area list page visible, all other pages will be edited later to represent grid-areas, was just copied from organization feature for now.